### PR TITLE
parse version string for comparison

### PR DESCRIPTION
**Pull request by sigmarkarl**
change version string comparison from

```python
if airflow.version >= "2.6.2"
```
to
```python
from packaging import version
if version.parse(airflow.version) >= version.parse("2.6.2")
```

# Jira Ticket

[BGD-5801](https://spotinst.atlassian.net/browse/BGD-5801)

# Demo

Run the ofas provider with airflow version 2.10+ and the connectsion options for Spark Connect wont show up

# Checklist:
- [x] I have filled relevant self assessment ([NodeJS](https://docs.google.com/forms/d/e/1FAIpQLSfl14u9AOBAmxVJ272tvO7XNuXE-EMvWaGcaRZalu1UAKB7RA/viewform), [Frontend](https://docs.google.com/forms/d/e/1FAIpQLSdiBPNKH81w_EkavihVL8Uwb0j7tP8PwJmLFYm2nCOQxz-1qw/viewform), [Backend](https://docs.google.com/forms/d/e/1FAIpQLSed_PsTJ5-XIWkFL6BSDE2AQRBVPmwc3PAmHZkUn-erzVI37Q/viewform?usp=sf_link))
- [x] I have run ESlint on my changes and fixed all warnings and errors (**NodeJS & Frontend Services**)
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have validated all the requirements in the Jira task were answered
- [x] I have all neccessary approvals for the design/mini design of this task
- [ ] I have approved the API changes and granular permission patterns (documentation subtask) (**For public services only**) 

[BGD-5801]: https://spotinst.atlassian.net/browse/BGD-5801?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ImpSy — DISMISSED** [review]

File: ocean_spark/connect/hook.py [hunk 1]
```diff
@@ -84,18 +84,18 @@ def get_app_page_url(self) -> str:
...
     def get_ui_field_behaviour() -> Dict:
         return {
-            "hidden_fields": ["extra", "schema"],
+            "hidden_fields": ["extra", "port"],
             "relabeling": {
                 "password": "API token",
                 "host": "Cluster id",
                 "login": "Account id",
-                "port": "Application Id",
+                "schema": "Application Id",
```
Comment (HafdisE) [inline ocean_spark/connect/hook.py:92]: I don't know anything about airflow, would using something called "schema" potentially have any unintended consequences or is it more or less up to us how to use those pre-existing fields?

Comment (HafdisE) [reply ocean_spark/connect/hook.py:92]: not that using port made sense either :D since that was fine I guess this will be fine.

Comment (sigmarkarl) [reply ocean_spark/connect/hook.py:92]: this "relabelling" approach is from the original OceanSparkHook, just fixing a bug here as port assumes an integer

**Review by HafdisE — APPROVED** [review]
LGTM

**Review by alextarasov-spot — APPROVED** [review]

**Pull request merged**